api/src/main/java/org/pmiops/workbench/ras/RasLinkService.java

```diff
@@ -128,7 +126,7 @@ public DbUser linkRasLoginGovAccount(String authCode, String redirectUrl) {
 
   /** Validates user has IAL2 setup. See class javadoc Step2 for more details. */
   static boolean isIal2(String acrClaim) {
-    return Arrays.asList(acrClaim.split(" ")).contains(ACR_CLAIM_IAL_2_IDENTIFIER);
+    return acrClaim.contains(ACR_CLAIM_IAL_2_IDENTIFIER);
```

Comment:
Did the acr format change? Is there an updated doc link we can add to this service file?

Reply:
The acr format does not change. I misunderstood that before. I didn't find standard doc to explains this and I feel ther is no standard for IAL status. What I know is RAS use this to represent IAL status, and the format is signing server `url + "/assurance/ial/2" ` I didn't ask if signing server url is always be the rasHost, but seems `/assurance/ial/2` is a good represent. So I just simplify this to see if that contains '/assurance/ial/2'